.circleci/config.yml

```diff
@@ -1,62 +1,97 @@
 defaults: &defaults
-  working_directory: /home/circleci/.go_workspace/src/github.com/gruntwork-io/terraform-aws-influx
-  machine:
-    image: circleci/classic:201711-01
-#  docker:
-#    - image: circleci/golang:1.10
+  machine: true
+  environment:
+    GRUNTWORK_INSTALLER_VERSION: v0.0.21
+    TERRATEST_LOG_PARSER_VERSION: v0.13.13
+    MODULE_CI_VERSION: v0.13.3
```

Comment:
Can you bump this to `v0.13.14`? It should be a straight upgrade process. The benefit is that this fixes a bug with the golang versions.